src/server/module-loader.ts

```diff
@@ -178,15 +183,15 @@ export class ModuleLoader extends EventTarget {
       if (basename(pathName).startsWith("_.context")) {
         if (isContextModule(endpoint)) {
           const loadContext = (path: string) => this.contextRegistry.find(path);
+          const openApi = this.openApiDocument;
 
           this.contextRegistry.update(
             directory,
-
             // @ts-expect-error TS says Context has no constructable signatures but that's not true?
-
             new endpoint.Context({
               loadContext,
-            }),
+              openApi
```

Comment:
Move this up one line so it's part of the object including `loadContext`. 

The idea to pass a single [parameter object](https://refactoring.guru/introduce-parameter-object) to the constructor for a Context class. We can use destructing to make the constructor's signature look like it takes a list of named arguments. 

Here's what a context class might look like. 


```js
export class Context {
   constructor({ loadContext, openApi }) {

     // grab the context object from some other path
     const otherContext = loadContext("/some/path");
    
     // read the OpenAPI document 
     const paths = openApi.paths;

     // ...
  } 
}
```